### ISSUE TITLE
Does the encode construction really use  the argument "eval_spatial_size" in the HybridEncoder's config in rtdetr_r50vd.yml?

### ISSUE THREAD

**Issue by heidi-zhenhai** [lyuwenyu/RT-DETR#83]
Hi,

Thank you very much for your release of PyTorch version.  I have one question about the HybridEncoder's setup in rtdetr_r50vd.yml.  
 HybridEncoder defines the following:
        ```
        eval_spatial_size: [640, 640]
       ```
From the constructor's signature, I can't find this argument (eval_spatial_size), instead I found `eval_size`.  So should I ignore eval_spatial_size, or I should change the name to eval_size?

Heidi

**Comment by lyuwenyu** [comment]
Yes, you are right.  I fix this typo in this commit https://github.com/lyuwenyu/RT-DETR/commit/8374002b848d4c2f123beb87cf8b120886684dce . You can pull lastest code and try it.

**Comment by heidi-zhenhai** [comment]
Thank you for the update.
